filtered-resource-service/src/main/java/uk/gov/gchq/palisade/service/filteredresource/web/AkkaHttpServer.java

```diff
@@ -31,6 +31,9 @@
 import java.util.function.Supplier;
 import java.util.stream.Collectors;
 
+/**
+ * Class to create an http server for akka
+ */
 public class AkkaHttpServer {
```

Comment:
I think that the Javas Doc in this class needs a bit more to describe the class and methods.